### ISSUE TITLE
Enum field not working

### ISSUE THREAD

**Issue by nyejon** [art1415926535/graphene-sqlalchemy-filter#19]
Hi

I am trying to filter on an Enum field, but if I add one to the filter I get:

```
File "/opt/venv/lib/python3.8/site-packages/graphene_sqlalchemy_filter/filters.py", line 531, in _generate_default_filters
fields = cls._generate_filter_fields(
File "/opt/venv/lib/python3.8/site-packages/graphene_sqlalchemy_filter/filters.py", line 610, in _generate_filter_fields
filter_field = field_type(description=doc)
TypeError: function() missing required argument 'code' (pos 1)
```

Do you have any ideas how to fix this?

**Comment by art1415926535** [comment]
Fixed in version [1.11.0](https://pypi.org/project/graphene-sqlalchemy-filter/1.11.0/).

Required graphene-sqlalchemy >= 2.2.0

**Comment by nyejon** [comment]
Fantastic! That was fast :) Tested and it's working
